docs/creating-manifest/visual-settings.md

```diff
@@ -673,67 +673,140 @@ values:
 @@!
 
 ### dockertags
-Field for displaying Docker tags within the list element.
+Field for displaying **Docker tags** within the *[list](#list)* element.
 
-![text](/img/dockertag.jpg)
+![text](/img/dockertags.png)
+
+The *tags* of specific *nodeType* can be displayed like in the wizard with no acquiring from the server:   
 @@@
 ```yaml
-name: Cloud Scripting
+type: install
+name: Dockertags aquiring
 
 settings:
   fields:
-  - type: dockertags
-    name: tag
-    values:
-    - name: latest
-    - name: first
-
-dockerImage:
-  name: jelastic/pokemongo-map
-  registry: ''
-  username: ''
-  password: ''
-
-env: {}
+    - type: dockertags      
+      nodeType: dockerengine
```

Comment:
use another nodeType to prevent confusion

Reply:
tomcat is used instead of dockerengine